core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -454,4 +454,14 @@ public static boolean validate() {
     }
     return valid;
   }
+
+  /**
+   * @return whether the configuration describes how to find the master host, either through
+   *         explicit configuration or through zookeeper
+   */
+  public static boolean masterHostConfigured() {
```

Comment:
I incline to put this method in `ConfigurationUtils` rather than `Configuration`. Leave this up to you.